doc/WinGetConfigure.md

```diff
@@ -0,0 +1,137 @@
+```
+
+In this case, you can respond with "Y" if you trust the PowerShell gallery to install this module.
+
+To verify the command succeeded you can run `Get-Module -listavailable PSDesiredStateConfiguration` and you should see the following output.
+
+```PowerShell
+PS C:\Users\denelon> Get-Module -listavailable PSDesiredStateConfiguration
+
+    Directory: C:\Users\denelon\Documents\PowerShell\Modules
+
+ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
+---------- -------    ---------- ----                                --------- ----------------
+Script     2.0.6                 PSDesiredStateConfiguration         Core      {Configuration, New-D…
+```
+
+### Install the PSDesiredStateConfiguration module
+
+Run `Install-Module PSDscResources` and you will be prompted with the familiar "Untrusted repository" where you may decide to trust the PowerShell Gallery.
+
+```PowerShell
+PS C:\Users\denelon> Get-Module -listavailable PSDscResources
+
+    Directory: C:\Users\denelon\Documents\PowerShell\Modules
+
+ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
+---------- -------    ---------- ----                                --------- ----------------
+Manifest   2.12.0.0              PSDscResources                      Desk
+```
+
+### Invoking DSC Resources
+The system is now in the proper state to install a PowerShell DSC Resource so you can run `Invoke-DSCResource`.
+
+**Note:** *We can ignore the `Import-DscResource -ModuleName PSDscResources` command since we're looking at `winget configure` and not the traditional PowerShell DSC v2 implementation.*
+
+In the example, we are setting a Process scoped environment variable. Before you run through the example you should validate the state of the environment variable outside of the context of PowerShell DSC as a form of independent verification that the desired state is being correctly modified. We will use `$ENV:DSC_EXAMPLE` to check for the presence, absence, and the value for the "DSC_EXAMPLE" environment variable.
+
+An empty response indicates the environment variable is absent. When the variable is present, the value is returned when you run `$ENV:DSC_EXAMPLE`.
+
+```
+PS C:\Users\denelon> $ENV:DSC_EXAMPLE
+Desired State Configuration
+```
+
+These examples are intentionally pedantic to ensure you have an understanding of the different behaviors exhibited when using `Invoke-DscResource`. When you are using DSC Resources to "get", "set", and "test" behaviors, you should independently verify the output. In this case you can run `$ENV:DSC_EXAMPLE` to verify the state of the Process scoped environment variable.
+
+ We will start with the example provided at [Invoking DSC Resources](https://learn.microsoft.com/powershell/dsc/getting-started/invoking-dsc-resources?view=dsc-2.0#invoking-dsc-resources). 
+
+ To remove the Process scoped environment variable run:
```

Comment:
This snippet seems to test if it is present, not remove it?

Reply:
Great catch! I added the test for the absence and the set for the removal.

Reply:
Done